src/DssSpell.sol

```diff
@@ -57,105 +44,192 @@ contract DssSpellAction is DssAction {
     //
     uint256 constant THREE_PT_FIVE_PCT  = 1000000001090862085746321732;
 
-    // Math
-    uint256 constant MILLION = 10**6;
-    uint256 constant WAD     = 10**18;
-    uint256 constant RAD     = 10**45;
-
     // Addresses
-    address constant RWA002_OPERATOR         = 0x2474F297214E5d96Ba4C81986A9F0e5C260f445D;
-    address constant RWA002_GEM              = 0xAAA760c2027817169D7C8DB0DC61A2fb4c19AC23;
-    address constant MCD_JOIN_RWA002_A       = 0xe72C7e90bc26c11d45dBeE736F0acf57fC5B7152;
-    address constant RWA002_A_URN            = 0x225B3da5BE762Ee52B182157E67BeA0b31968163;
-    address constant RWA002_A_INPUT_CONDUIT  = 0x2474F297214E5d96Ba4C81986A9F0e5C260f445D;
-    address constant RWA002_A_OUTPUT_CONDUIT = 0x2474F297214E5d96Ba4C81986A9F0e5C260f445D;
+    address constant MCD_DOG              = 0x135954d155898D42C90D2a57824C690e0c7BEf1B;
+    address constant MCD_END              = 0xFB4713C20CCD81c53BC55e12923d37a4f0DB3E93;
+    address constant MCD_ESM              = 0x0f8B44Df587Abe2533cE7D75cB004C127Fe1a267;
+    address constant ILK_REGISTRY         = address(0);
+    address constant CLIPPER_MOM          = 0x79FBDF16b366DFb14F66cE4Ac2815Ca7296405A0;
+    address constant MCD_CLIP_LINK_A      = 0x832Dd5f17B30078a5E46Fdb8130A68cBc4a74dC0;
+    address constant MCD_CLIP_CALC_LINK_A = 0x7B1696677107E48B152e9Bf400293e98B7D86Eb1;
```

Comment:
```
0x7B1696677107E48B152e9Bf400293e98B7D86Eb1
└─ ward: 0xDA0FaB0700A4389F6E6679aBAb1692B4601ce9bf
   └─ eoa: user
```

Reply:
I've already fixed it. Relied the pause proxy and denied the deployer. Please verify it.